tests/network/primary_pod_network.go

```diff
@@ -144,24 +143,21 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 })
 
 func vmiWithDefaultBinding() *v1.VirtualMachineInstance {
```

Comment:
please correct me if I'm wrong, but `vmiWithDefaultBinding` implies that there is a Binding involved, however `NewCirros` has no MasqueradeBinding nor BridgeBinding configured in it.

Reply:
Would `vmiWithImplicitBinding` work? When no binding is selected, it falls back to the implicit default

Reply:
nah.. it'll make it more vague :) let's leave it with that.